source/uwp/Renderer/lib/MediaHelpers.cpp

```diff
@@ -370,8 +372,23 @@ HRESULT HandleMediaClick(
         EventRegistrationToken mediaOpenedToken;
         THROW_IF_FAILED(mediaElement->add_MediaOpened(Callback<IRoutedEventHandler>([=](IInspectable* /*sender*/, IRoutedEventArgs* /*args*/) -> HRESULT
         {
+            boolean audioOnly;
+            RETURN_IF_FAILED(localMediaElement->get_IsAudioOnly(&audioOnly));
+
+            ComPtr<IImageSource> posterSource;
+            RETURN_IF_FAILED(localMediaElement->get_PosterSource(&posterSource));
+
+            if (audioOnly && posterSource == nullptr)
+            {
+                // If this is audio only and there's no poster, set the height so that the 
+                // controls are visible.
+                ComPtr<IFrameworkElement> mediaAsUiElement;
+                RETURN_IF_FAILED(localMediaElement.As(&mediaAsUiElement));
+                RETURN_IF_FAILED(mediaAsUiElement->put_Height(c_audioHeight));
```

Comment:
Don't we already have a mediaAsUiElement variable above that we could use?

Reply:
This variable is misnamed, it's actually a FrameworkElement. Updated to the correct name.

---
In reply to: [202800736](https://github.com/Microsoft/AdaptiveCards/pull/1713#discussion_r202800736) [](ancestors = 202800736)